application-mocca-calendar-ui/src/main/resources/MoccaCalendar/Macro.xml

```diff
@@ -665,9 +657,41 @@ if (typeof(XWiki) == "undefined" || typeof(XWiki.widgets) == "undefined" || type
   }) // end PlainList extends View
 
   jQuery.fullCalendar.views.plainList = PlainList;
-  
 } // if the XWiki.widgets.ModalPopup  is defined
-  
+
+});
+
+/**
+ * Deleting an event
+ */
+require(['jquery'], function($) {
+  /**
+  * Events triggered before deleteEvents modal is shown: save the button that triggers
+  * the modal, close MoccaCalendarPopup, update the name of event on displayed text.
+  */
+  $(document).on('show.bs.modal', '#deleteEvents', function(event) {
+    var relatedTarget = $(event.relatedTarget);
+    $(this).data('relatedTarget', $(relatedTarget));
+    var calendarPopup = $(relatedTarget).data('calendarPopup');
+    calendarPopup.closeDialog();
+    var eventName = '«' + calendarPopup.interactionParameters.event.title + '»';
+    $(this).find('.modal-body').text(
+      "$services.localization.render('MoccaCalendar.calendarevent.confirmDelete', '__replace__')"
```

Comment:
```suggestion
      $jsontool.serialize($services.localization.render('MoccaCalendar.calendarevent.confirmDelete', '__replace__'))
```
If you want to be 100% safe (the translation text may contain special characters like quote, apostrophe or new lines,  and we don't always control / verify that).